### PR TITLE
Adding Github templates for pull requests, issues and contributing.

### PR DESCRIPTION
**Pull request by mariozaizar**
### Description

Adds [Github Templates](https://github.com/blog/2111-issue-and-pull-request-templates) for pull requests, issues and contributing.

- CONTRIBUTING.md
- .github/ISSUE_TEMPLATE.md
- .github/PULL_REQUEST_TEMPLATE.md

### CODE REVIEW

**Comment by thilgen** [comment]
@zendesk/strongbad

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+## Issues
+
```
Comment (grosser) [inline .github/ISSUE_TEMPLATE.md]: Empty template ?

File: TODO.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# TODO
+
+- Write usage instructions here
```
Comment (grosser) [inline TODO.md]: just delete ... nobody is going to do it :D

**Comment by grosser** [comment]
:+1:

**Comment by ggrossman** [comment]
@mariozaizar rebase this against master and the Travis build should work properly.

**Pull request merged**